app/components/ui/SectionTitle.scss

```
@@ -0,0 +1,40 @@
+@use '~/assets/styles/_variables.scss' as *;
```

Stylelint (17.6.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)